src/convertToObject.js

```diff
@@ -6,7 +6,22 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  if (!sourceString.length) {
+    return {};
+  }
+
+  const stringToObject = sourceString
+    .split(';')
+    .map((value) => value.split(':').map((spaces) => spaces.trim()))
+    .reduce((styleObject, key) => {
+      if (key[0] && key[1]) {
+        styleObject[key[0]] = key[1];
+      }
+
+      return styleObject;
+    }, {});
+
+  return stringToObject;
```

Comment:
The function `convertToObject` is well-named and the logic is mostly correct. However, there's a minor issue with the `reduce` method. The parameter names in the callback function could be more descriptive to enhance readability. Instead of `key`, which is actually an array containing both the property name and value, consider using a name like `stylePair` or `declaration`. Additionally, within the `reduce` callback, the current parameter name `styleObject` could be renamed to `accumulator` or `styles` to better represent its role as the accumulator in the `reduce` process.